config/countries-partial.php

```diff
@@ -28,7 +28,7 @@
     ],
     CountryCode::IRELAND => [
         'systemLocale' => 'en_IE.UTF-8',
-        'locale' => 'ie',
+        'locale' => 'en',
```

Comment:
if we just use the 2 char approach yes I think you're right
![image](https://github.com/Pod-Point/countries/assets/117723846/f93e0128-8f5a-40e7-ac3d-07dbc036ca33)